fetch.sh

```diff
@@ -183,6 +185,24 @@ getwindowmanager () {
     fi
 }
 
+
+# Font (Configurable with "-F" and "--font" at launch)
+# Currently only supported for X Terminals
+if [ -f $HOME/.XResources ]; then
+    font=$(cat $HOME/.XResources | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+elif [ -f $HOME/.Xresources ]; then
+    font=$(cat $HOME/.Xresources | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+elif [ -f $HOME/.XDefaults ]; then
+    font=$(cat $HOME/.XDefaults | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+elif [ -f $HOME/.Xdefaults ]; then
+    font=$(cat $HOME/.Xdefaults | grep -v ! | awk '/font/ {print $2}' | sed 's/xft://;s/:pixelsize.*$//')
+else
+    font="Unknown"
+fi
+
+# Screen Resolution (Configurable with "-R" and "--resolution" at launch)
+resolution=$(xdpyinfo | awk '/dimensions:/ {print $2}')
```

Comment:
If users would like these details hidden by default, it would be simpler to have a list at the top of the file listing all possible options then a variable containing what the user would like to display. For example:

    valid=( img os kernel uptime pkgs shell wm cpu mem song)
    display=( img os shell wm song )

to be used in conjunction with show or hide flags, to save the user effort overall.